_posts/2020-12-28-changelog-138.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #138"
+date: 2020-12-28 00:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6530 -->
+* Enable `Build` tool window by default. The `Build` tool window provides structured output for `build`, `run` and `test` commands.
+  Note, this feature requires Cargo +1.48.0
+
+  {% include gif-img.html path="/assets/posts/changelog-138/build-tool-window" w="700px" %}
+
+  You can disable it via `org.rust.cargo.build.window` option in `Experimental Features` dialog (`Help | Find Action` and type `Experimental features` to open the dialog)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6419 -->
+* Provide experimental code completion ranking for Rust based on machine learning.
+  It applies rules learned from data we have gathered anonymously during our EAPs, which results in better suggestions.
+  Note that we have not collected any source code, only information about your interactions with the code completion UI.
+  The ranking model is disabled by default for now.
+  You can enable it via `Editor | General | Code Completion | Machine Learning-Assisted Completion` [settings](https://www.jetbrains.com/help/idea/auto-completing-code.html#ml_completion).
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6443 -->
+* Update obsolete references in `use` groups via `Move` refactoring (<kbd>F6</kbd>).
+  Now, when moving `foo` from `mod1` to `mod2`, the outdated group import `use mod1::{foo, bar};` will be replaced with the proper `use mod2::foo; use mod1::bar;`.
+
+  {% include gif-img.html path="/assets/posts/changelog-138/move-use" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5644 -->
+* Add `Fill missing arguments` quick-fix to complete a function call arguments with default values (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-138/fill-arguments" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5771 -->
+* Add the intention to change item visibility (by [@Kobzol])
+
+  {% include gif-img.html path="/assets/posts/changelog-138/make-pub" w="700px" %}
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6390 -->
+* Introduce experimental `Cargo.toml` dependency completion based on the new local Cargo registry index.
+  Note, the corresponding feature is disabled by default for now. You can enable it via `org.rust.crates.local.index` option in `Experimental Features` dialog (`Help | Find Action` and type `Experimental features` to open the dialog)
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6492 -->
+* Show [quick documentation](https://www.jetbrains.com/help/idea/viewing-reference-information.html#inline-quick-documentation) for [keyword](https://doc.rust-lang.org/stable/std/#keywords) like elements like `fn`, `enum`, `async`, etc
```

Comment:
```suggestion
* Show [quick documentation](https://www.jetbrains.com/help/idea/viewing-reference-information.html#inline-quick-documentation) for [keyword](https://doc.rust-lang.org/stable/std/#keywords)-like elements, e.g. `fn`, `enum`, `async`, etc
```
BTW not sure if we really want "keyword-like elements" here and not just "keywords". Probably such exactness is redundant for users